polaris-react/src/components/IndexTable/IndexTable.scss

```
@@ -784,14 +784,21 @@ $loading-panel-height: 53px;
 
 .Table-unselectable {
   // stylelint-disable-next-line selector-max-class, selector-max-specificity -- generated by polaris-migrator DO NOT COPY
-  &.Table-sticky .TableCell:first-child {
-    left: 0;
-    background-color: var(--p-color-bg-surface);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    z-index: var(--pc-index-table-sticky-cell);
-    position: sticky;
-  }
+  &.Table-sticky {
+    .TableCell:first-child {
+      left: 0;
+      background-color: var(--p-color-bg-surface);
+      // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
+      z-index: var(--pc-index-table-sticky-cell);
+      position: sticky;
+    }
 
+    .TableRow-subheader {
+      .TableCell:first-child {
+        background-color: var(--p-color-bg-surface-secondary);
+      }
+    }
+  }
   .TableRow-hovered {
```

Noticed there's another bug where hovering disabled rows in an unselectable table gives the first cell a hover state. Could you fix that in your PR too?

```suggestion
  .TableRow-hovered:not(.TableRow-disabled {
```

---

Sure! Fixed)